sdk/nodejs/Makefile

```diff
@@ -28,7 +28,8 @@ build_package:: ensure
 	cp tests/runtime/jsClosureCases_8.js bin/tests/runtime
 	cp tests/runtime/jsClosureCases_10_4.js bin/tests/runtime
 	cp -R tests/automation/data/. bin/tests/automation/data/
-	cp README.md ../../LICENSE package.json ./dist/* bin/
+	cp README.md ../../LICENSE ./dist/* bin/
+	sed 's/"devDependencies"/"_devDependencies"/g' package.json > bin/package.json
```

Comment:
This feels a bit awkward - both removing all `devDependencies` and leaving `_devDependencies` in the file. Would there be a way to remove just the `mockPackage` line?

Reply:
ok. just pushed another solution to strip lines containing "mockpackage".